todoman/cli.py

```diff
@@ -532,7 +532,8 @@ def move(ctx, list, ids):
 @click.option('--grep', help='Only show tasks with message containg TEXT')
 @click.option(
     '--sort',
-    help='Sort tasks using these fields',
+    help=('Sort tasks using fields like "start", "due", "priority", "created_at", "percent_complete" etc.'
+          '(for all fields see todoman/docs/source/usage.rst)'),
```

Comment:
I would use the link `https://todoman.readthedocs.io/en/stable/usage.html`

Reply:
I think we can also programatically generate a list of sortable fields.
But it's fine as-is right now, we can improve on that later. 🙂